Gopkg.toml

```diff
@@ -41,6 +41,10 @@
   name = "github.com/golang/protobuf"
   version = "1.1.0"
 
+[[constraint]]
```

Comment:
why do we need this specific version?

Reply:
I guess we don't, but it's the latest, and also the version that was already included by one of our dependencies. We also seem to be pinning the rest of our dependencies to specific versions, so it seemed consistent. 🤷‍♂

Reply:
Sure I'll pin it to the latest major with a caret 👍